### PR TITLE
Add mmperf

### PR DESCRIPTION
**Pull request by mariecwhite**

### CODE REVIEW

File: .github/workflows/run_mmperf.yml
Comment (mariecwhite) [inline .github/workflows/run_mmperf.yml:14]: Remove before merge.

File: microbenchmarks/mmperf/build_mmperf.sh
Comment (pzread) [inline microbenchmarks/mmperf/build_mmperf.sh:1]: From an offline discussion, LGTM if we move this into an experimental folder for now

**Review by pzread — APPROVED** [review]

**Pull request merged**
